lib/brains/tests/test_webhook.py

```diff
@@ -0,0 +1,55 @@
+from django.core.urlresolvers import reverse
+
+from braintree.webhook_notification import WebhookNotification
+from braintree.webhook_notification_gateway import WebhookNotificationGateway
+from nose.tools import eq_
+
+from lib.brains.tests.base import BraintreeTest
+
+
+def parsed(**kwargs):
```

Comment:
``parsed_notice`` would make more sense to me. I was confused when first reading this code.